src/custom-element.js

```diff
@@ -1434,10 +1447,16 @@ function createBaseAmpElementProto(win) {
  */
 export function registerElement(win, name, implementationClass) {
   knownElements[name] = implementationClass;
+  const klass = createBaseCustomElementClass(win, name);
 
-  win.document.registerElement(name, {
-    prototype: createAmpElementProto(win, name),
-  });
+  const supportsCustomElementsV1 = 'customElements' in win;
+  if (supportsCustomElementsV1) {
+    win['customElements'].define(name, klass);
```

Comment:
IIUC, adding an extern would define `window.customElements`:

> ...you declare the extern by writing JavaScript that defines the symbol you want preserved, and then giving that JavaScript to the Closure Compiler. [[source]](https://developers.google.com/closure/compiler/docs/api-tutorial3#externs)

Seems like this would break `custom-elements` on browsers that don't support V1.